src/components/CustomProperties/CustomProperties.tsx

```diff
@@ -8,6 +8,11 @@ export const DEFAULT_COLOR_SCHEME: ColorScheme = 'light';
 
 export const STYLE_SHEET_ID = 'polaris-custom-properties';
 
+const canUseDOM = Boolean(
```

Comment:
Use the isServer utility perhaps? https://github.com/Shopify/polaris-react/blob/main/src/utilities/target.ts#L1

Reply:
[86d1314](https://github.com/Shopify/polaris-react/pull/5299/commits/86d1314854ba73684dfd57d2fdf9f336d3566738)